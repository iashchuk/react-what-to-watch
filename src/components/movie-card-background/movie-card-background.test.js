import React from "react";
import renderer from "react-test-renderer";
import MovieCardBackground from "./movie-card-background.jsx";

describe(`MovieCardBackground`, () => {
  it(`renders correctly`, () => {
    const component = renderer.create(<MovieCardBackground />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
