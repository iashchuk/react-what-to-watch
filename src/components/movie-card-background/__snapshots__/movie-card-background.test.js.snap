// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardBackground renders correctly 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="The Grand Budapest Hotel"
    src="img/bg-the-grand-budapest-hotel.jpg"
  />
</div>
`;
